common/lib/capa/capa/capa_problem.py

```diff
@@ -562,7 +562,12 @@ def find_question_label(self, answer_id):
         """
         _ = get_gettext(self.capa_system.i18n)
         # Some questions define a prompt with this format:   >>This is a prompt<<
-        prompt = self.problem_data[answer_id].get('label')
+        log_error = False
```

Comment:
This is a temporary log, which will give us information and unblock the course team.